### ISSUE TITLE
bug: YouTube Revanced App is crashing while switching Accounts

### ISSUE THREAD

**Issue by udoxi** [revanced/revanced-patches#1091]
### Type

Error at runtime

### Bug description

Still there
YouTube Revanced App (Non Root) is crashing while switching accounts and also crashing at first start-up

Patch version `2.118.1`
YouTube version `17.43.36`

### Steps to reproduce

while switching accounts and also crashing at first start-up

### Relevant log output

```shell
Nothing
```

### Screenshots or videos

https://user-images.githubusercontent.com/115417493/203469292-a5d698da-d7ff-4377-9391-287b42c7da94.mp4

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by pungorandras** [comment]
java.lang.VerifyError: Verifier rejected class aaef: void aaef.h(aadx, aadw, java.util.List, long, long, boolean) failed to verify: void aaef.h(aadx, aadw, java.util.List, long, long, boolean): [0xAF] invalid use of move-result* (declaration of 'aaef' appears in /data/app/~~YhTuR6087QlUMYGOFxd32Q==/app.revanced.android.youtube-YX4QL7NPKQRXDjUtXxt3ig==/base.apk!classes3.dex)
	at aaed.c(PG:5)
	at esj.run(PG:138)
	at tpg.run(PG:7)
	at oeu.run(PG:5)
	at oey.run(PG:23)
	at nnm.run(PG:13)
	at com.google.android.apps.youtube.embeddedplayer.service.jar.client.q.run(PG:15)
	at java.lang.Thread.run(Thread.java:1012)

**Comment by pungorandras** [comment]
It happens in another point than earlier.

**Comment by rohan2000github** [comment]
https://user-images.githubusercontent.com/62375786/203489729-3d9254e4-bd02-4cc0-bf7d-877ded191f14.mp4

For me Revanced crashing when switching accounts , when turning wifi on/off and when tapping the downloads tab as u can see in the video.
Patch version 2.118.1

**Comment by YouSummoner** [comment]
I think I found it, it's something related to the hide-info-cards patch, if the app is patched without it, the app doesn't crash while trying to download a video and opening the downloads section...🧐

**Comment by pungorandras** [comment]
nice finding

**Comment by rohan2000github** [comment]
Yeah hide-info-card patch is the guilty here, all crashes ( account switching crash, wifi on/off crash, download tab open crash) are solved if exclude this patch.

**Comment by YouSummoner** [comment]
Heyyyy, actually another issue, even after excluding the hide-info-cards patch, the videos download but don't show up in the downloads section, it's getting soo annoying now, please fix this urgently, first priority...😶

**Comment by YouSummoner** [comment]
Exclude, zoom-haptics, startup-shorts-player, and swipe-controls patches too, and the downloads issue will be fixed, again, soo annoying...

**Comment by oSumAtrIX** [comment]
Please check if this is solved.

**Comment by rks2000** [comment]
> Please check if this is solved.

Tried it in latest patch and youtube version 17.45.36 ,
 Crashes are still there.

**Comment by pungorandras** [comment]
Unfortunately not
